include/internal/quic_fifd.h

```diff
@@ -72,13 +73,15 @@ int ossl_quic_fifd_init(QUIC_FIFD *fifd,
                         void (*sstream_updated)(uint64_t stream_id,
                                                 void *arg),
                         void *sstream_updated_arg,
-                        QLOG *qlog);
+                        QLOG *(*get_qlog_cb)(void *arg),
+                        void *get_qlog_cb_arg);
```

Comment:
What has qlog got do do with the PR?

Reply:
We were generating empty qlog files if we created an SSL object and then didn't actually use it to make a connection. This caused the CI test for qlog to balk at the empty file as a sign something is wrong. This is in turn caused by the fact that our test is skipped in the blocking invocation of the multistream test, since it means the SSL objects are setup and then torn down without being used. So this fix is necessary for this PR to pass CI.